pkg/staticpod/staticpod.go

```diff
@@ -336,9 +336,9 @@ func addVolumes(p *v1.Pod, src []string, volume typeVolume) {
 }
 
 func addExtraMounts(p *v1.Pod, extraMounts []string) {
-	var sourceType v1.HostPathType
 
 	for i, rawMount := range extraMounts {
+		var sourceType v1.HostPathType = v1.HostPathUnset // Reset sourceType for each mount
```

Comment:
Technically you don't have to initialize it with a specific value, `v1.HostPathType` is just a `string` and `v1.HostPathUnset` is an empty string. So just moving the initialization inside the loop would be fine.
```suggestion
		var sourceType v1.HostPathType
```

Reply:
Thank you for reviewing it. I have updated it.